specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -457,6 +471,35 @@ model Evaluation {
   systemData?: Record<string>;
 }
 
+@doc("Evaluation Group Definition")
```

Comment:
Should we support custom aggregation in future?

We should document default aggregations that come out of box.